feat: JS performance docs (#2133)
getsentry/sentry-docs: src/platforms/javascript/common/config/performance.mdx

          name: location.pathname
            .replace(/\d+/g, "<digits>")
            .replace(/[a-f0-9]{32}/g, "<hash>"),
        };
      },
    }),
  ],
  tracesSampleRate: 1.0, // Be sure to lower this in production
});
```

### Filter out unwanted Spans

In the `BrowserTracing` options we provide a method called `shouldCreateSpanForRequest`. This function can be used to filter our unwanted Spans like XHR's running health checks or something similar. By default `shouldCreateSpanForRequest` is already filtering out what was defined in `tracingOrigins`.

```javascript
import * as Sentry from "@sentry/browser";
import { Integrations } from "@sentry/tracing";

Sentry.init({
  dsn: "___PUBLIC_DSN___",
  integrations: [
    new Integrations.BrowserTracing({
      shouldCreateSpanForRequest: url => {
        // Example of filter out spans that contain `health`
        if (url.match(/health/)) { 
            return false;
        }
        return true;
      },
    }),
  ],
  tracesSampleRate: 1.0, // Be sure to lower this in production
});
```

### Manual Instrumentation

To manually instrument certain regions of your code, you can create a transaction to capture them.

> This is valid for both JavaScript Browser and Node and works independently of the `BrowserTracing` integration.

Reviewer: ```suggestion
This is valid for both the JavaScript Browser and Node SDKS, and works independently of the `BrowserTracing` integration.
```

```javascript
const transaction = Sentry.startTransaction({ name: "test-transaction" });
const span = transaction.startChild({ op: "functionX" }); // This function returns a Span
// functionCallX
span.finish(); // Remember that only finished spans will be sent with the transaction
transaction.finish(); // Finishing the transaction will send it to Sentry
```

> Here is a different example. If you want to create a transaction for a user interaction on your page, you need to do the following:

Reviewer: ```suggestion
This example creates a transaction for a user interaction on your page:
```
